src/initialise.js

```diff
@@ -50,9 +50,9 @@ module.exports = async app => {
 	 * Tako should be a private (also know as internal) GitHub App, installed on the same account it is administered by.
 	 *
 	 * @see https://developer.github.com/apps/managing-github-apps/making-a-github-app-public-or-private/#private-installation-flow
-	 * @see https://octokit.github.io/rest.js/#api-Apps-getInstallations
+	 * @see https://octokit.github.io/rest.js/#api-Apps-listInstallations
 	 */
-	const installations = await octokit.apps.getInstallations().catch(err => {
+	const installations = await octokit.apps.listInstallations().catch(err => {
```

Comment:
Oh does this work?

I noticed that the old method is deprecated in `v15.18.0`, but wasn't sure that the new one would work.

Looks like the build failed because of these changes.

Maybe worth holding off until the next major version of Probot, if there is one.

Reply:
Nice change 👍